nerves_toolchain_aarch64_nerves_linux_gnu/mix.exs

```diff
@@ -48,7 +48,7 @@ defmodule NervesToolchainAarch64NervesLinuxGnu.MixProject do
   defp deps do
     [
       {:nerves, "~> 1.4", runtime: false},
-      {:nerves_toolchain_ctng, "~> 1.9.0", runtime: false}
+      {:nerves_toolchain_ctng, path: "../nerves_toolchain_ctng", runtime: false}
```

Comment:
Is this intentional to change back to a path dependency? (Many of these toolchains were changed)

Reply:
Yes, there's one commit that only makes that change. It's needed so that crosstool-ng gets updated. Without it, `nerves_toolchain_ctng` would get pulled from hex.pm and we'd get the old version.